perf_test/k8s/wml-serving-keras-predictor.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: Predictor
+metadata:
+  name: keras-predictor
+spec:
+  modelType:
+    name: keras
+  path: fvt/tensorflow/keras-mnist/mnist.h5
```

Comment:
The path does not exit on the minio server installed by the modelmesh-serving
Should be `path: keras/mnist.h5`

Reply:
The model type and path that we support in public repo the moment: https://github.com/kserve/modelmesh-serving/tree/main/config/example-predictors

Reply:
https://github.com/kserve/modelmesh-serving/tree/main/config/example-predictors doesn't have predictors:
- mleap
- pmml
- snapmlmnist
- spark
Where shall I get their path from or shall I leave them as is?

Reply:
Maybe we should drop them ? because no one can actually use them. Those model frameworks are not supported by the modelmesh-serving at the moment and the provided Minio Server doesn't have those either.

Reply:
Maybe we should drop them ? because no one can actually use them. Those model frameworks are not supported by the modelmesh-serving at the moment and the provided Minio Server doesn't have those either.

Reply:
I agree. Maybe also consider to remove wml from file names to be more generic?